### PR TITLE
Added LF ACT (another open source tooling site)

### PR DESCRIPTION
**Pull request by winterrocks**
One more source for open source tooling added

### CODE REVIEW

**Comment by anajsana** [comment]
Thanks for your contribution @winterrocks !

Do you want to add this within the "open source tooling" section? If so, It seems you need to add an additional space to fix the indentation. Right now, the render markdown file shows as:
Screenshot 2022-04-20 at 10-32-40 Try markmap

Could you please commit this change? 🙂

File: ospo-mindmap/Content/ospomindmap.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ development team of the Linux Project
   - Adopt existing tools
       - Commercial tools
       - [Open Source tools](https://oss-compliance-tooling.org/Tooling-Landscape/OSS-Based-License-Compliance-Tools/)
+      - [Linux Foundation's Automate Compliance Tooling](https://automatecompliance.org/)
```
Comment (anajsana) [inline ospo-mindmap/Content/ospomindmap.md:78]: fix indentation to show as part of the open source tooling section:

```
  - Adopt existing tools
      - Commercial tools
      - [Open Source tools](https://oss-compliance-tooling.org/Tooling-Landscape/OSS-Based-License-Compliance-Tools/)
             - [Linux Foundation's Automate Compliance Tooling](https://automatecompliance.org/)
```

**Review by anajsana — CHANGES_REQUESTED** [review]

File: ospo-mindmap/Content/ospomindmap.md (hunk 1 above)
Comment (winterrocks) [reply ospo-mindmap/Content/ospomindmap.md:78]: yes, it is better under Open Source tools. 
Thanks Ana!

**Comment by winterrocks** [comment]
Yes, better under Open Source tools.

**Pull request closed without merging**